test/unit/annotation_layer_spec.js

```diff
@@ -172,4 +188,35 @@ describe('Annotation layer', function() {
       expect(borderStyle.verticalCornerRadius).toEqual(0);
     });
   });
+
+  describe('FileAttachmentAnnotation', function() {
+    var loadingTask;
+    var annotations;
+
+    beforeEach(function() {
+      var pdfUrl = combineUrl(window.location.href,
+                              '../pdfs/annotation-fileattachment.pdf');
+      loadingTask = PDFJS.getDocument(pdfUrl);
+      waitsForPromiseResolved(loadingTask.promise, function(pdfDocument) {
```

Comment:
It's really strange, but that does not work either. This is the result:

`TEST-UNEXPECTED-FAIL | should correctly parse a file attachment | TypeError: ann
otations is undefined in http://127.0.0.1:63625/test/unit/annotation_layer_spec.
js (line 215)`